### ISSUE TITLE
Formatted human readable output.ofx

### ISSUE THREAD

**Issue by archont00** [kedder/ofxstatement#63]
Would it be possible to format the output.ofx with indents and new-lines? Like what "xmllint --format" does.

```
<OFX><SIGNONMSGSRSV1><SONRS><STATUS><CODE>0</CODE><SEVERITY>INFO</SEVERITY></STATUS><DTSERVER>20170616224548</DTSERVER>
```
would be:
```
<OFX>
  <SIGNONMSGSRSV1>
    <SONRS>
      <STATUS>
        <CODE>0</CODE>
        <SEVERITY>INFO</SEVERITY>
      </STATUS>
      <DTSERVER>20170616224548</DTSERVER>
... cut on purpose ...
```
Easier to read and debug.

**Comment by kedder** [comment]
Umm, probably. I usually use `ofxdump` to debug files though.

**Comment by BIZ4Africa** [comment]
+1 on  this request.
I use Odoo accounting software, it rejects OFX from ofxstatement based on this issue.
I always need "sed" the ouput file from ofxstatement to make it acceptable.

**Comment by kedder** [comment]
@BIZ4Africa interesting. Can you share your sed script?

**Comment by kedder** [comment]
Released with `ofxstatement-0.8.0`
